src/systemctl/systemctl.c

```diff
@@ -8574,8 +8574,6 @@ int main(int argc, char*argv[]) {
         case ACTION_RUNLEVEL4:
         case ACTION_RUNLEVEL5:
         case ACTION_RESCUE:
-        case ACTION_EMERGENCY:
-        case ACTION_DEFAULT:
```

Comment:
instead of dropping this without replacement, I think it would be good, adding it in a comment to some later location in this switch statement, and adding to that comment why we don#t care for these actions here. Otherwise the next time somebody looks at this statement he might think we forgot something there..

Reply:
So `start_with_fallback()` works correctly even for actions without a fallback.  In that sense it's fine to have these 2 cases.  But still, they're never used.

There are... several different subsets of possible actions, depending where it's used.  It definitely confused me :).  Used here, it already implicitly excludes four actions:

        ACTION_EXIT,
        ACTION_SUSPEND,
        ACTION_HIBERNATE,
        ACTION_HYBRID_SLEEP,

It is easier to understand if the unused actions are dropped, so the switch correctly reflects `systemctl` v.s. all the legacy commands.

You might have a point about this needing to be more explicit though.  I might have stared at this too long by now, to consider what it looks like to a new reader :).  I'm sending a more verbose version, as you suggest.